eng/scripts/automation/test/GenerateAndBuildLib-functions.tests.ps1

```diff
@@ -0,0 +1,109 @@
+#Requires -Version 7.0
+
+. (Join-Path $PSScriptRoot ".." ".." ".." "common" "scripts" "Helpers" PSModule-Helpers.ps1)
+
+Install-ModuleIfNotInstalled "Pester" "5.3.3" | Import-Module
+
+BeforeAll {
+    . $PSScriptRoot/../GenerateAndBuildLib.ps1
+}
+
+Describe "update autorest.md" -Tag "Unit" {
+    BeforeAll {
+        copy-item -Path $PSScriptRoot/autorest.test.md -Destination $PSScriptRoot/autorest.md
+    }
+
+    it("set mutilple input-file") {
+        $inputfiles = "https://github.com/Azure/azure-rest-api-specs/blob/b2bddfe2e59b5b14e559e0433b6e6d057bcff95d/specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2019-11-01-preview/account.json;https://github.com/Azure/azure-rest-api-specs/blob/1424fc4a1f82af852a626c6ab6d1d296b5fe4df1/specification/purview/data-plane/Azure.Analytics.Purview.MetadataPolicies/preview/2021-07-01/purviewMetadataPolicy.json"
+
+        $inputfile = ""
+        $fileArray = $inputfiles.Split(";")
+        if (($inputfiles -ne "") -And ($fileArray.Length -gt 0)) {
+            for ($i = 0; $i -lt $fileArray.Count ; $i++) {
+                $inputfile = $inputfile + [Environment]::NewLine + "- " + $fileArray[$i]
+            }
+        }
+        CreateOrUpdateAutorestConfigFile -autorestFilePath $PSScriptRoot/autorest.md -namespace "Azure.Purview.Account" -inputfile "$inputfile"
+
+        $autorestConfigYaml = Get-Content -Path $PSScriptRoot/autorest.md
+        $range = ($autorestConfigYaml | Select-String -Pattern '```').LineNumber
+        if ( $range.count -gt 1) {
+            $startNum = $range[0];
+            $lines = $range[1] - $range[0] - 1
+            $autorestConfigYaml = ($autorestConfigYaml | Select -Skip $startNum | Select -First $lines) |Out-String
+        }
+        Install-Module -Name powershell-yaml -Force -Verbose -Scope CurrentUser
```

Comment:
Should we do this Install-Module/Import once at the top of the script?

Reply:
Also the underlying script imports it as well, so it may not be necessary?

Reply:
updated